src/nvim/strings.c

```diff
@@ -1528,3 +1528,45 @@ char_u *reverse_text(char_u *s)
 
   return rev;
 }
+
+/// Replace all occurrences of 'what' with 'rep' in 'src'. If no replacement happens then NULL is
```

Comment:
I think it's better to use double quotes or backticks here instead of single quotes.